sequencer/src/lib.rs

```diff
@@ -203,16 +239,23 @@ pub enum Error {
 
 #[allow(clippy::too_many_arguments)]
 async fn init_hotshot<N: network::Type>(
-    nodes_pub_keys: Vec<PubKey>,
-    known_nodes_with_stake: Vec<<PubKey as SignatureKey>::StakeTableEntry>,
     node_id: usize,
     priv_key: PrivKey,
     networks: Networks<SeqTypes, Node<N>>,
     config: HotShotConfig<PubKey, ElectionConfig>,
     metrics: &dyn Metrics,
-    instance_state: &NodeState,
+    instance_state: NodeState,
```

Comment:
I trust there's a good reason to take ownership here.

Reply:
Yeah, the type of `HotShotInitializer::from_genesis` changed so it in turn now takes ownership